pnpxai/explainers/deep_lift_shap.py

```diff
@@ -0,0 +1,32 @@
+from typing import Dict, Tuple, Callable
+
+import torch
+import shap
+import captum
+from torch import Tensor
+from pnpxai.explainers.base import Explainer
+from torch.nn.modules import Module
+from captum.attr import DeepLiftShap as CaptumDeepLiftShap
+from pnpxai.explainers.base import Explainer
+from pnpxai.explainers.types import ForwardArgumentExtractor
+from pnpxai.explainers.utils.baselines import BaselineMethodOrFunction, BaselineFunction
+from pnpxai.utils import format_into_tuple, format_out_tuple_if_single
+
+
+class DeepLiftShap(Explainer):
```

Comment:
Could you, please, add appropriate supported modules like this:

```suggestion
class DeepLiftShap(Explainer):
    SUPPORTED_MODULES = [Convolution]
```

Reply:
Thank you for the correction. Please check the latest commit.